src/applications/rated-disabilities/components/MVIError.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { getAppUrl } from '@department-of-veterans-affairs/platform-utilities/exports';
+
+const facilityLocatorUrl = getAppUrl('facilities');
+
+export default function MVIError() {
```

Comment:
Need to talk to Robin later and determine if this is actually needed or if I can just rely on the `verify` prop on the `RequiredLoginView` component